lib/Conversion/TorchToMhlo/Basic.cpp

```diff
@@ -1376,23 +1391,24 @@ void mlir::torch::torch_to_mhlo::populateBasicOpPatternsAndLegality(
   target.addIllegalOp<RuntimeAssertOp>();
   patterns.add<ConvertRuntimeAssertOp>(typeConverter, context);
 
-#define INSERT_UNARY_PATTERN(AtenOp, MhloOp)                                   \
+#define INSERT_UNARY_PATTERN(AtenOp, StablehloOp)                              \
   target.addIllegalOp<AtenOp>();                                               \
-  patterns.add<ConvertAtenUnaryOp<AtenOp, MhloOp>>(typeConverter, context)
+  patterns.add<ConvertAtenUnaryOp<AtenOp, StablehloOp>>(typeConverter, context)
   INSERT_UNARY_PATTERN(AtenCloneOp, mhlo::CopyOp);
```

Comment:
Nit: how about changing `mhlo::CopyOp` to `stablehlo::ConvertOp` or `tensor::CastOp`, since `AtenCloneOp` is a no-op exactly in HLO? So we can remove all dependencies on MHLO in the TorchToMhlo pass.

Reply:
Thanks!  Replaced with `stablehlo::ConvertOp`.